tutorials/standalone-tensorflow-raspberry-pi/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Setting Up a Standalone Tensorflow Prediction Device with Raspberry Pi and Google Cloud
+description: Learn how to set up a standalone Tensorflow server in Raspberry Pi with a model trained in Cloud ML Engine.
+author: arieljassan
+tags: Tensorflow, Raspberry Pi, MLEngine
+date_published: 2018-08-19
+---
+
+
+# Setting Up a Standalone Tensorflow Prediction Device with Raspberry Pi and Google Cloud
+
+This tutorial describes how to set up a Tensorflow server in a Raspberry Pi 3B and deploy on it a Tensorflow saved model. 
+
+The architecture of a machine learning model trained in the cloud and served locally is particularly useful when the IoT device has poor connectivity or when the serving availability is critical. 
+
+This tutorial assumes you have a Raspberry Pi with SD card and Raspbian installed on it, that you are familiar with Python and Tensorflow and that you have available a Tensorflow saved model. If you don’t have access to a saved model, you can alternatively use the saved model resulting from the Image Classification using Flowers dataset tutorial that is provided in this guide for download.
+
+## Challenges solved
```

Comment:
I'll clean these up in the production pass if you don't get to it, but note that our publishing system requires a blank line between headings and other items.

Reply:
done.